tests/test_dask_image/test_ndfourier/test_core.py

```diff
@@ -218,15 +218,28 @@ def test_fourier_filter_non_positive(funcname, s):
         "fourier_uniform",
     ]
 )
-def test_fourier_filter(funcname, s):
+@pytest.mark.parametrize(
+    "n, axis",
+    [
+        ('real', -1),
```

Comment:
It's a bit confusing that n is supposed to be an integer, but here you have it as a string & then later redefine the string to an integer. It might be clearer to have the pytest parameter be a string all the time, and then define n in the body of the test itself.

Reply:
good point. I have updated it in adb14ca